frontend/lib/resolution/scope-queries.cpp

```diff
@@ -877,7 +877,12 @@ doResolveUseStmt(Context* context, const Use* use,
           kind = VisibilitySymbols::CONTENTS_EXCEPT;
           // check that we do not have 'except A as B'
           for (const AstNode* e : clause->limitations()) {
-            assert(e->isIdentifier());
+            // If we just assert this condition directly, and Chapel is
+            // compiled without assertions, we get unused variable warnings.
+            // This otherwise redundant if condition avoids that possibility.
+            if (!e->isIdentifier()) {
```

Comment:
This is good here, but just in case you haven't seen it yet, sometimes you can write
```C++
(void) myUnusedVariable;
```
To get the unused variable warnings off your back.